Microsoft.Research/RegressionTest/ClousotTests/Options.cs

```diff
@@ -4,45 +4,46 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using ClousotTests;
 
 namespace Tests
 {
+    [Flags]
+    public enum TestRuns
+    {
+        None = 0,
+        V40AgainstV35Contracts = 1,
+        V40 = 2
+    }
+
     public class Options
     {
         private const string RelativeRoot = @"..\..\..\..\..\";
         internal const string TestHarnessDirectory = @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug";
-        private static readonly string RootDirectory;
+        protected static readonly string RootDirectory = Path.GetFullPath(RelativeRoot);
 
-        static Options()
-        {
-            RootDirectory = Path.GetFullPath(RelativeRoot);
-        }
+        private readonly string _outDirectory;
```

Comment:
I guess we can take combined approach like @sharwell: convert a bunch of project using StyleCopAnalyzers and then try to clean up the code (add more comments, kill a dead code, enhance names etc) as we work on them.